common/dynamicconfig/config/testConfig.yaml

```diff
@@ -1,4 +1,4 @@
-frontend.validSearchAttributes:
+frontend.namespaceCount:
```

Comment:
what is this change?

Reply:
This is some random config setting used in config tests. I changed it some existent key because `validSearchAtributes` are not there anymore.